docs/server-side/v2/foundation/webhook.mdx

```diff
@@ -108,6 +108,7 @@ This event will be send when peer leaves the room
 | role       | `string`     | Role of the user                                   | host                                 |
 | left_at    | `timestamp`  | Timestamp when user left                           | 2020-11-11T17:32:17Z                 |
 | duration   | `number`     | Duration the user spent in the room in seconds     | 36000                                |
+| reason     | `string`     | User leave reason                                  | client request                       |
```

Comment:
Description - Reason for the peer leaving, see more details below

Reply:
fixed